_rules/bypass-blocks-cf77f2.md

```diff
@@ -0,0 +1,279 @@
+---
+id: cf77f2
+name: Bypass Blocks of Content
+rule_type: composite
+description: |
+  This rule checks that each page has a mechanism to bypass repeated blocks of content.
+accessibility_requirements:
+  wcag20:2.4.1: # Bypass Blocks (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_rules:
+  - 047fe0
+  - 7b576d
+  - 8a213c
+  - b40fd1
+  - 3e12e1
+  - e53727
+acknowledgements:
+  authors:
+    - Jean-Yves Moyen
+---
+
+## Applicability
+
+This rule applies to any [HTML web page][].
+
+## Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+
+- [Document has headings][]; or
+- [Document has a main landmark][]; or
```

Comment:
These two techniques don't supply a mechanism in my opinion. They supply categories that AT might use make use of but if a keyboard user doesn't have AT then there is no mechanism to skip past all the repeating content.

Reply:
I see the point, but:

---
Both [H69](https://www.w3.org/WAI/WCAG21/Techniques/html/H69) and [ARIA11](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA11) mention:
> This technique relates to:
Success Criterion 2.4.1: Bypass Blocks (Sufficient as a way to meet an unwritten technique)

Which indicates that these techniques (and rules) should be enough for WCAG. 

---
Additionally, heading navigation is not only for AT. According to https://www.456bereastreet.com/archive/201003/heading_navigation_in_web_browsers/ Opera and some Firefox extensions can do it (or could, this is an old page). It is arguable whether a browser extension is AT or not, and the frontier between UA and AT can be blurry. So we may consider that as "AT features in browsers".

But the point is that both (HTML) headings and landmarks are in the DOM and can be used. So it's not something that is purely specific to, say, screen readers.

---
There is an Accessibility Support note in the composite rule that points out that while these techniques are sufficient to pass 2.4.1, they are not the best way to do so.

---
**Proposal**:
Would it be OK for you if I add a strong Accessibility Support in the corresponding rules in the line of "identifying blocks is only useful if the UA/AT combination can use it"?
(and strengthen the one in the composite rule accordingly)

Reply:
Yeah. Those assumptions would solve this issue for me! Thank you!